README.md

```diff
@@ -47,9 +48,13 @@ A curated list of tools that will help you when working with or building on-top
 - [HBase Java](https://github.com/GoogleCloudPlatform/cloud-bigtable-client) - Official Java libraries and HBase client extensions for accessing Google Cloud Bigtable.
 - [Go](https://github.com/googleapis/google-cloud-go/tree/master/bigtable) - Official implementation of the Google Cloud Bigtable Go client.
 - [PHP](https://github.com/googleapis/google-cloud-php-bigtable) - Official implementation of the Google Cloud Bigtable PHP client.
+
+### Other Client-Libraries
 - [Simple Bigtable](https://github.com/spotify/simple-bigtable) - Java based client built and maintained by Spotify.
 - [Rust Bigtable](https://github.com/durch/rust-bigtable) - Rust library for working with Google Bigtable Data API.
 - [AsyncBigtable](https://github.com/OpenTSDB/asyncbigtable) - Implementation of AsyncHBase but on top of Google's Cloud Bigtable service.
+- [node-hbase-client](https://github.com/alibaba/node-hbase-client) - Asynchronous HBase client for Node.js, pure JavaScript implementation.
```

Comment:
I don't know how the Node HBase client library works under the hood; if it's a wrapper around the Java HBase client library, then it would be usable with Bigtable as well (as long as it uses HBase 1.0 API or later). Otherwise, it may not be compatible with Bigtable.

To be on the safe side, I would remove these Node.js libraries unless there are some docs or examples somewhere that talk about how to connect them to Bigtable.

Here are the docs on this:

* https://cloud.google.com/bigtable/docs/hbase-bigtable
* https://cloud.google.com/bigtable/docs/reference/libraries#client-libraries-usage-hbase-java